### PR TITLE
Set default release to 29 when running on non-fedora hosts

### PR DESCRIPTION
**Pull request by andrunko**
https://github.com/debarshiray/toolbox/issues/131

### CODE REVIEW

File: toolbox [hunk 1]
```diff
@@ -1259,7 +1267,12 @@ usage()
 
 arguments=$(save_positional_parameters "$@")
 
-release_default=$(get_host_version_id)
+host_id=$(get_host_id)
+if [ "$host_id" = "fedora" ]; then
```
Comment (debarshiray) [inline toolbox]: Nitpick: would be nice to redirect the standard error to file descriptor 3 for `--verbose` output.

Comment (debarshiray) [reply toolbox]: I took the liberty to tweak it as above.

**Comment by andrunko** [comment]
@debarshiray thanks for checking/merging - I guess we can close issue #131 now and open another issue if/when we plan to change the default image based on the host.

**Comment by debarshiray** [comment]
Ok, sure.

**Pull request merged**